pennylane/ops/op_math/prod.py

```diff
@@ -281,6 +281,17 @@ def overlapping_ops(self) -> List[Tuple[Wires, List[Operator]]]:
     def has_decomposition(self):
         return True
 
+    @property
+    def obs(self):
+        r"""Access the operands of a ``Prod`` instance"""
+        # This is temporary property to smoothen the transition to the new operator arithmetic system.
+        # In particular, now that the __matmul__ (@ python operator) method between operators generates Prod instead of Tensor instances.
```

Comment:
```suggestion
        # In particular, the __matmul__ (@ python operator) method between operators now generates Prod instead of Tensor instances.
```